spring-cloud-dataflow-server/src/main/resources/application.yml

```diff
@@ -37,6 +37,9 @@ info:
 #     config:
 #        discovery:
 #           enabled: true
+#     dataflow:
```

Comment:
This file is totally defunc even as an example. No need to update it. We should probably zap all commented lines.